.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and Push
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The Ansible Chatbot Stack version to build and push"
+        required: true
+        default: "0.2.9"
+
+jobs:
+
+  build_push:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    env:
+      ANSIBLE_CHATBOT_VERSION: ${{ github.event.inputs.version }}
+
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: network=host
+
+      - name: Login to Quay
+        uses: docker/login-action@v3
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: refs/heads/main
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+
+      - name: Environment Setup
+        run: |
+          make setup
+
+      - name: Base Ansible Chatbot Stack Build
+        run: |
+          export PYPI_VERSION=${{ env.ANSIBLE_CHATBOT_VERSION }}
```

Comment:
This makes `llama stack build` use the `llama-stack` version defined in `PYPI_VERSION` i.e. `0.2.8`, `0.2.9` etc.

I think this is different from `ANSIBLE_CHATBOT_VERSION` that could be `0.0.1`, `0.0.2` etc. No?